config/taxonomy/Makefile

```diff
@@ -0,0 +1,10 @@
+ROOT_DIR := ../..
+all: clean compile gen
```

Comment:
None of these targets exist in this makefile

Reply:
Changed it.